docs/content/how-to/gcp/configure-image-registry.md

```
@@ -0,0 +1,251 @@
+
+### Check ClusterOperator Status
+
+Verify the image registry operator is available:
+
+```bash
+KUBECONFIG=hosted-kubeconfig oc get clusteroperator image-registry
+```
+
+Expected output:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)